subsys/dm/dm.c

```diff
@@ -391,7 +395,12 @@ static void dm_thread(void)
 				if (dm_context.ranging_status) {
 					nrf_dm_populate_report(&report);
 					nrf_dm_calc(&report);
-					process_data(&report);
+#ifdef CONFIG_NRF_DM_USE_HIGH_PRECISION_CALC
+					high_precision_estimate = nrf_dm_high_precision_calc(&report);
```

Comment:
On nrf53832 when calling this funcjion, MPU Fault occurs.
Additionally, for nrf53833 and nrf528322, the stack for dm_thread needs to be increased.